jira/client.py

```diff
@@ -3149,6 +3152,34 @@ def move_to_backlog(self, issue_keys):
             raise NotImplementedError('No API for moving issues to backlog for agile_rest_path="%s"' %
                                       self._options['agile_rest_path'])
 
+    def map_custom_python_name(self, customfieldname=None, pythonfieldname=None):
+        python_to_custom = self.field_map
+        custom_to_python = [(v,k) for k,v in python_to_custom]
+
+        try:
+            if customfieldname is not None:
+                return list_lookup(custom_to_python, customfieldname)
+            elif pythonfieldname is not None:
+                return list_lookup(python_to_custom, pythonfieldname)
+            else:
+                raise KeyError("Need to provide a custom field name, or a python field name.")
+        except:
```

Comment:
I raise exception on line 3165 for forcing a jump into the except block to return the default key (the  key to look up), since anything that is not a custom field (and thus in the mapping lists) should remain untouched, since those are normal fields. Thoughts on how to do this more cleanly?